files/opensesame.spec

```
@@ -161,6 +179,13 @@ type = unsigned_short
 description = maximum value measured during calibration
 type = unsigned_short
 
+[sensors/device]
+description = path of the sensor device
+default = "/dev/ttyACM0"
+
+[sensors/log]
+description = logging sensor data to file
+default = "/home/olimex/data.log"
```

Does /dev/null work?

```suggestion
default = "/home/olimex/data.log"
example = "/dev/null"
```

---

We removed `sensors/log` because you said we shouldn't write to the SD card. Do you think we should implement the logging?

---

Then please also remove from spec.

> Do you think we should implement the logging?

Can also be implemented later, not relevant for #75.